packages/jumpstarter-driver-flashers/oci_bundles/buildroot/build_fits.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+dnf install --setopt=install_weak_deps=false -y git make gcc gcc-c++ which file diffutils wget cpio rsync bc lzop zip patch perl tar qemu-system-aarch64 qemu-img unzboot uboot-tools kmod awk
+
+git clone --depth 1 --branch 2025.05  https://github.com/buildroot/buildroot /buildroot
+
+# build default buildroot kernel & initramfs
+cp buildroot_defconfig  /buildroot/configs/
+cp -R overlay /buildroot
+( cd /buildroot; make buildroot_defconfig && make )
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)